e2e-polybft/property_test.go

```diff
@@ -0,0 +1,57 @@
+package e2e
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/types"
+	"pgregory.net/rapid"
+)
+
+// property based tests enabled
+const envPropertyBaseTestEnabled = "E2E_PROPERTY_TESTS"
```

Comment:
Should we move this to TestCluster ([here](https://github.com/0xPolygon/polygon-edge/blob/feature/EVM-353/e2e-polybft/framework/test-cluster.go#L26) for example), so that all E2E testing environment vars are kept on a single place?

Reply:
https://github.com/0xPolygon/polygon-edge/pull/1149/commits/47178fb01acb738db85dab15f573b2ceebc50c0a